modules/mod_ginger_base/mod_ginger_base.erl

```diff
@@ -8,7 +8,7 @@
 -mod_description("Ginger Base").
 -mod_prio(250).
 -mod_depends([mod_content_groups, mod_acl_user_groups]).
--mod_schema(7).
+-mod_schema(9).
```

Comment:
@DorienD He's pulling your leg. Module downgrades are prohibited so you end up bumping the version number several times during development unless you want to mess with your DB.